control-station/src/services/PodSocketClient.ts

```diff
@@ -15,7 +15,7 @@ export enum State {
 interface ServerToClientEvents {
 	connect: () => void;
 	disconnect: (reason: Socket.DisconnectReason) => void;
-	serverResponse: (data: string) => void;
+	serverResponse: (data: PodData) => void;
```

Comment:
Chore: Not sure why TypeScript didn't catch this, but we'll also need this to be `Partial`